src/app/static/webpack.config.js

```diff
@@ -70,6 +64,26 @@ module.exports = {
     ]
 };
 
+const appConfig = {...commonConfig,
+    entry: './src/app/index.ts',
+    output: {
+        path: path.resolve(__dirname, './public/js'),
+        publicPath: '/public/js/',
+        filename: 'app.js'
+    }
+};
+
+const forgotPasswordAppConfig = {...commonConfig,
+    entry: './src/app/forgotPassword.ts',
+    output: {
+        path: path.resolve(__dirname, './public/js'),
+        publicPath: '/public/js/',
+        filename: 'forgotPassword.js'
+    }
+};
+
+module.exports = [appConfig, forgotPasswordAppConfig];
+
 if (process.env.NODE_ENV === 'production') {
```

Comment:
Cool that this multiple bundle thing was so simple to configure! But now that an array is being returned instead of a single object do these production optional properties have to be applied explicitly to each item in the array? (realise we aren't yet setting this flag either, just added a ticket so we don't forget https://vimc.myjetbrains.com/youtrack/issue/mrc-425)

Reply:
Good point!